File: day-1/python/even-odd.py
# convert the given number to binary if it is even or
# hexadecimal if it is not


def evens_and_odds(n):
    if n % 2 == 0:
        return bin(n).replace("0b", "")
    else:
        return hex(n).replace("0x", "")


num = int(input("Enter an Integer Number: ").strip())
print(evens_and_odds(num))

File: day-1/php/even-odd.php
<?php

// convert the given number to binary if it is even or
// hexadecimal if it is not

function evens_and_odds($n)
{
    if ($n % 2 == 0) {
        return decbin($n);
    } else {
        return dechex($n);
    }
}

$num = 10;
echo evens_and_odds($num);

echo "\n";

$num = 13;
echo evens_and_odds($num);
